server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -225,6 +227,38 @@ export class DeliLambdaFactory
 					closeType === LambdaCloseType.ActivityTimeout ||
 					closeType === LambdaCloseType.Error
 				) {
+					if (document?.isEphemeralContainer) {
```

Comment:
Now that Deli will be making HTTP requests to the summary endpoint, we might need to check if there is anything else needed from the point of view of security/compliance. I think we are already running LinkerD within the Deli pods, but if not, we might need to do that

Reply:
We already have HTTP calls that we are making to other microservices like riddler. So, this is not a new change to the that behavior. But just to be sure, I made sure that we are running linkerd in deli

![image](https://github.com/microsoft/FluidFramework/assets/448106/20ebcc88-1a8a-424f-b545-619e7375ed35)

Reply:
Thanks for confirming! 😄